tools/ci.sh

```diff
@@ -229,6 +229,7 @@ function ci_powerpc_setup {
 }
 
 function ci_powerpc_build {
+    make ${MAKEOPTS} -C ports/powerpc submodules
```

Comment:
Should we be adding the "submodules" target to a Makefile dependency rather than calling it here?